styles.css

```diff
@@ -0,0 +1,83 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+@media screen and (max-width: 768px) {
+  body {
+    background-color: #e5e5e5;
+    padding: 0;
+    margin: 0;
+  }
+
+  .hero {
+    background-image: url(assets/images/background-shapes.png);
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  main {
+    background-color: #fff;
+    width: 100vw;
+    height: 100vh;
+    border-radius: 0 0 0 15%;
+  }
+
+  header {
+    padding-top: 16px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  nav {
+    padding-right: 32px;
+  }
+
+  .my-logo {
```

Comment:
- [ ] kindly consider selecting the logo well so that the color you gave it will be applied  because that will make it look exactly like the one in Figma
- hint 👍 
```
.my-logo a {
}
```